Source/WebCore/platform/audio/gstreamer/PlatformRawAudioDataGStreamer.cpp

```diff
@@ -77,6 +78,11 @@ static std::pair<GstAudioFormat, GstAudioLayout> convertAudioSampleFormatToGStre
     return { GST_AUDIO_FORMAT_UNKNOWN, GST_AUDIO_LAYOUT_INTERLEAVED };
 }
 
+Ref<PlatformRawAudioData> PlatformRawAudioData::create(Ref<MediaSample>&& sample)
+{
+    return PlatformRawAudioDataGStreamer::create(GRefPtr { sample->platformSample() });
```

Comment:
```suggestion
    return PlatformRawAudioDataGStreamer::create(GRefPtr { sample->platformSample() });
    return PlatformRawAudioDataGStreamer::create(GRefPtr(sample->platformSample().sample.gstSample));
```

Reply:
yeah, I had made that change on my linux box, but it didn't make it back to the Mac :(